src/integration-test/java/uk/gov/hmcts/juror/api/moj/report/grouped/ReasonableAdjustmentsReportITest.java

```diff
@@ -0,0 +1,536 @@
+package uk.gov.hmcts.juror.api.moj.report.grouped;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.web.client.TestRestTemplate;
+import org.springframework.test.context.jdbc.Sql;
+import uk.gov.hmcts.juror.api.moj.controller.reports.request.StandardReportRequest;
+import uk.gov.hmcts.juror.api.moj.controller.reports.response.AbstractReportResponse;
+import uk.gov.hmcts.juror.api.moj.controller.reports.response.GroupByResponse;
+import uk.gov.hmcts.juror.api.moj.controller.reports.response.GroupedReportResponse;
+import uk.gov.hmcts.juror.api.moj.controller.reports.response.GroupedTableData;
+import uk.gov.hmcts.juror.api.moj.controller.reports.response.StandardReportResponse;
+import uk.gov.hmcts.juror.api.moj.report.AbstractGroupedReportControllerITest;
+import uk.gov.hmcts.juror.api.moj.report.DataType;
+import uk.gov.hmcts.juror.api.moj.report.ReportHashMap;
+import uk.gov.hmcts.juror.api.moj.report.ReportLinkedMap;
+
+import java.time.LocalDate;
+import java.util.List;
+
+@Sql({
+    "/db/truncate.sql",
+    "/db/mod/truncate.sql",
+    "/db/administration/createUsers.sql",
+    "/db/mod/reports/ReasonableAdjustmentsReportITest_typical.sql"
+})
+@SuppressWarnings("PMD.LawOfDemeter")
+class ReasonableAdjustmentsReportITest extends AbstractGroupedReportControllerITest {
+
+    @Autowired
+    public ReasonableAdjustmentsReportITest(TestRestTemplate template) {
+        super(template, ReasonableAdjustmentsReport.class);
+    }
+
+    @Override
+    protected String getValidJwt() {
+        return getCourtJwt("415");
+    }
+
+    @Override
+    protected StandardReportRequest getValidPayload() {
+        return addReportType(StandardReportRequest.builder()
+            .fromDate(LocalDate.of(2024, 1, 1))
+            .toDate(LocalDate.of(2024, 1, 2))
+            .build());
+    }
+
+    @Test
+    @SuppressWarnings({
+        "PMD.JUnitTestsShouldIncludeAssert"//False positive
+    })
+    void positiveTypicalCourt() {
+        testBuilder()
+            .jwt(getCourtJwt("415"))
```

Comment:
Not needed as if you do not specify a JWT this will default to the value from protected String getValidJwt() {...}